site/_includes/layouts/namespace-reference.njk

```diff
@@ -81,7 +81,16 @@
                 </div>
               </li>
             {% endif %}
-
+            {% if feature.manifestRequirements.length %}
+              <li>
+                <div class="code-sections__label">Manifest Keys</div>
+                <div>
+                  {% for manifestKey in feature.manifestRequirements %}
+                    <code>{{ manifestKey }}</code><br />
+                  {% endfor %}
+                </div>
```

Comment:
I feel like the current copy may be a little too terse. For the sake of experimentation, I tried another presentation. WDYT?

> ![Screen Shot 2021-08-10 at 5 05 50 PM](https://user-images.githubusercontent.com/1047763/128950371-767b35c7-4456-4238-a80e-c9ba8c6c0294.png)

Offhand I'm not sure what other APIs would display this message, so I don't know if the suggested copy is accurate.